src/constants/metagameEventType.ts

```diff
@@ -0,0 +1,57 @@
+export enum MetagameEventType {
```

Comment:
Maybe we should add a map that contains the event id as key and then a enum of zones that can be used to get zones based on an event(maybe go one step further and add other properties like duration and faction). Also it can be used to keep track of new events that are added to game as they are not part of the map.

Reply:
If we add a "initiated by" feature it'll be frontend and we'll be able to figure it out from event ID really

Reply:
@microwavekonijn I've re-read this a few times and don't fully get it, are you suggesting creating a map / array full of the duruations (which I agree is a good idea) but what was the last part?

Reply:
Basically `Map<event_id_type, {zoneId: number, duration: number, etc}>`. Then we can quickly lookup an event id, if it exists then create a new metagame event instance with the info from the map, else report that an unkown event was triggered.